paasta_tools/cli/cmds/spark_run.py

```diff
@@ -497,7 +494,6 @@ def get_docker_run_cmd(
         cmd.append("--volume=%s" % volume)
     cmd.append("%s" % docker_img)
     cmd.extend(("sh", "-c", docker_cmd))
-    cmd.append(sensitive_env)
```

Comment:
how did this even work before?

Reply:
`execlpe` expected last argument as an environment as dictionary. Looks like the only main env variables which mattered the spawned process are the aws creds. This was appended as last element of the list `docker_run_cmd`

Reply:
tl;dr: Sensitive env variables a.k.s aws creds were skipped to be added as part of the actual command (visible in logs). 
But passed as part of env variables.
I am still keeping the behavior of skipping adding them to `--env` the same.